triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -211,6 +212,17 @@ private[trigger] object ServiceConfig {
           + JdbcConfig.help()
       )
 
+    opt[Boolean]("db-table-prefix")
+      .action { (b, c) =>
+        c.copy(dbTablePrefix = if (b) { "trigger_service_" }
```

Comment:
Would it be worth going all the way and letting the user choose the prefix? (Would it be wise?)

Reply:
I'm not sure I like that from a UX perspective. Having two different defaults for the same app seems confusing.

> By default, we do not add a prefix to tables. IF you want, you can turn this flag to add the default prefix of `trigger_service_` to every table. You can also pass a second flag (only if you passed the first) to change the default prefix (which is disabled by default).

Seems a bit clunky to explain, whereas

> Pass this flag to set a prefix on all table names. Example: `--flag trigger_service_`.

seems much simpler and lets people achieve exactly the same thing.

Reply:
Sure, that's fine with me, thanks.